src/config.ts

```diff
@@ -19,56 +19,250 @@ export const DEFAULT_CONFIG = `
+                return 0;
+            }
+        });
+        return files[0];
+    }
+
+    /**
+     * Creates a default config in the given workspace folder.
+     * The config is created under `.mirrord/mirrord.json`.
+     * @param folder workspace folder for the config
+     * @returns path to the created config
+     */
+    private static async createDefaultConfig(folder: vscode.WorkspaceFolder): Promise<vscode.Uri> {
+        let path = vscode.Uri.joinPath(folder.uri, ".mirrord", "mirrord.json");
+        await vscode.workspace.fs.writeFile(path, Buffer.from(DEFAULT_CONFIG));
+        return path;
+    }
+
+    /**
+     * Handles `mirrord.changeSettings` command.
+     * Allows the user to open a mirrord config file selected from quick pick.
+     * Quick pick options in order:
+     *  - active config (if set)
+     *  - configs used in launch configurations across the workspace
+     *  - default configs across the workspace
+     */
+    public async changeSettings() {
+        const options: Map<string, vscode.Uri> = new Map();
+
+        // Active config first.
+        if (this.active) {
+            options.set(`(active) ${vscode.workspace.asRelativePath(this.active.path)}`, this.active);
+        }
+
+        // Then all configs found in launch configurations across the workspace.
+        const folders = vscode.workspace.workspaceFolders || [];
+        for (const folder of folders) {
+            let launchConfigs = vscode.workspace.getConfiguration("launch", folder)?.get<LaunchConfig[]>("configurations") || [];
+            for (const launchConfig of launchConfigs) {
+                let rawPath = launchConfig.env?.["MIRRORD_CONFIG_FILE"];
+                if (!rawPath) {
+                    continue;
+                }
+
+                let path;
+                if (rawPath.startsWith("/")) {
+                    path = vscode.Uri.file(rawPath);
+                } else {
+                    path = vscode.Uri.joinPath(folder.uri, rawPath);
+
+                }
+
+                if (folders.length > 1) {
+                    options.set(`(launch config ${folder.name}:${launchConfig.name}) ${vscode.workspace.asRelativePath(path)}`, path);
+                } else {
+                    options.set(`(launch config ${launchConfig.name}) ${vscode.workspace.asRelativePath(path)}`, path);
+                }
+            }
+        }
+
+        // Then all default configurations across the workspace.
+        for (const folder of folders) {
+            let path = await MirrordConfigManager.getDefaultConfig(folder);
+            if (path) {
+                options.set(`(default) ${vscode.workspace.asRelativePath(path)}`, path);
+            } else {
+                let path = vscode.Uri.joinPath(folder.uri, ".mirrord/mirrord.json");
+                options.set(`(create default) ${vscode.workspace.asRelativePath(path)}`, path);
+            }
+        }
+
+        let quickPickOptions = [...options.keys()];
```

Comment:
What if there's just one option or no options? Can't we just open it immediately? Same as we do today

Reply:
Nice catch, thanks